charts/cray-spire/Chart.yaml

```diff
@@ -23,7 +23,7 @@
 ---
 apiVersion: v2
 name: cray-spire
-version: 1.5.7
+version: 1.5.6
```

Comment:
Why did this version go backwards?

Reply:
Oops Must have been working too fast. Fixed now.